lib/utils.ts

```diff
@@ -8,11 +8,15 @@ export function cn(...inputs: ClassValue[]) {
 }
 
 export function getInitials(name: string) {
-  return name
-    .split(" ")
-    .slice(0, 2)
-    .map((word) => word[0].toUpperCase())
-    .join("");
+  if (!name.trim()) return "";
+
+  const words = name.trim().split(" ");
```

Comment:
Why not putting the ".toUpperCase()" method here?
```suggestion
  const words = name.trim().toUpperCase().split(" ");
```